### PR TITLE
Clarified the traces_sample_rate behavior in Python.

### PR DESCRIPTION
**Pull request by antonpirker**
Some clarifying on how `traces_sample_rate` behaves in Python.

Fixes https://github.com/getsentry/sentry-python/issues/2486

### CODE REVIEW

File: src/platforms/common/configuration/options.mdx [hunk 1]
```diff
@@ -933,6 +933,13 @@ A boolean value, if true, transactions and trace data will be generated and capt
 
 A number between 0 and 1, controlling the percentage chance a given transaction will be sent to Sentry. (0 represents 0% while 1 represents 100%.) Applies equally to all transactions created in the app. Either this or <PlatformIdentifier name="traces-sampler" /> must be defined to enable tracing.
 
+<PlatformSection supported={["python"]}>
+If <PlatformIdentifier name="traces-sample-rate" /> is 0 this means that no new traces will be created. But if you have another service (for example an JS frontend) that makes requests to your service that include trace information, that traces will be continued and thus transactions will be sent to Sentry.
```
Comment (sentrivana) [inline src/platforms/common/configuration/options.mdx:937]:
```suggestion
If <PlatformIdentifier name="traces-sample-rate" /> is 0, this means that no new traces will be created. However, if you have another service (for example a JS frontend) that makes requests to your service that include trace information, those traces will be continued and thus transactions will be sent to Sentry.
```

File: src/platforms/common/configuration/options.mdx [hunk 2]
```diff
@@ -933,6 +933,13 @@ A boolean value, if true, transactions and trace data will be generated and capt
...
+
+If you want to disable tracing for good you need to set <PlatformIdentifier name="traces-sample-rate" />`=None` or set <PlatformIdentifier name="enable-tracing" />`=False`. In this case now new traces will be started and no incoming traces will be continued.
```
Comment (sentrivana) [inline src/platforms/common/configuration/options.mdx:939]:
```suggestion
If you want to disable tracing for good you need to set <PlatformIdentifier name="traces-sample-rate" />`=None` or set <PlatformIdentifier name="enable-tracing" />`=False`. In this case no new traces will be started and no incoming traces will be continued.
```

**Review by sentrivana — APPROVED** [review]
Thanks for adding this! Left some suggestions.

**Comment by sl0thentr0py** [comment]
should probably be universal and not just for python

File: src/platforms/common/configuration/options.mdx (hunk 2 above)
Comment (shanamatthews) [reply src/platforms/common/configuration/options.mdx:939]: Made a small tweak to @sentrivana 's suggestion.

**Review by shanamatthews — APPROVED** [review]
Looks good - thanks for the good suggestions @sentrivana !

**Pull request merged**